src/sentry/plugins/providers/integration_repository.py

```diff
@@ -145,7 +139,7 @@ def create_repository(
                 except IntegrationError:
```

Comment:
Are we confident we can ignore this? What's the scenario in which this happens, and why can can we ignore it?

Reply:
good question. i haven't actually touched this code but it looks like an error that's raised when something goes wrong fetching anything to do with an integration